src/server.js

```diff
@@ -53,10 +54,22 @@ module.exports = function() {
 			log.error("Path to SSL certificate is invalid. Stopping server...");
 			process.exit();
 		}
-		server = server.createServer({
-			key: fs.readFileSync(keyPath),
-			cert: fs.readFileSync(certPath)
-		}, app);
+		if (config.https.ca.length && fs.existsSync(caPath)) {
+			log.error("Path to SSL ca bundle is invalid. Stopping server...");
+			process.exit();
+		}
+		if (config.https.ca.length) {
```

Comment:
Would it be dreadful to restructure this to do:

```
server = server.createServer({
  key: fs.readFileSync(keyPath),
  cert: fs.readFileSync(certPath),
  ca:  caPath ? fs.readFileSync(caPath) : undefined
}, app);
```

That should? work exactly the same (test it), but it also removes the duplicate code, and looks much nicer, imo.